### PR TITLE
IDE-2756 fix fragment deploying error

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

File: com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java [hunk 1]
```diff
@@ -169,80 +170,91 @@ private void installOrUpdate(File outputFile) throws Exception {
 			isFragment = fragmentHost != null;
 
 			bsn = bundle.getBsn();
+
+			if(isFragment) {
+				hostBSN =
+						new Parameters(fragmentHost).keySet().iterator().next();
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: There may be multiple parameters in the hostBSN so we can't asssume the "host" bundle will be the first parameter.

Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: Do we perform this in the IDE?  where we rrefresh the host on deploy of a fragment?  If so can we just check the code there?  Perhaps it is incorrect as well.

File: com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java [hunk 2]
```diff
@@ -169,80 +170,91 @@ private void installOrUpdate(File outputFile) throws Exception {
...
-			if (hostId > 0) {
-				_blade.out().println("refreshing host " + hostId);
-
-				response = client.send("refresh " + hostId);
-
-				_blade.out().println(response);
-			}
 		}
 
-		client.close();
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: Why did you remove the client.close()?

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java (hunk 1 above)
Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: in our ide side code , we just use 
`ragmentHostName.trim().substring( 0, fragmentHostName.indexOf( ";bundle-version" ) );`
to get the hostBSN, so actually it is the same principle.
but I can't think of any parameter that could be the first one before hostBSN,

File: com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java (hunk 2 above)
Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: hey Greg, I didn't remove this line , see right side line 241

File: com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java (hunk 1 above)
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: Is there another API on Parameters that could give us what we want>

Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: hey fist , there should not be multiple parameters in Fragment-Host attribute , see https://github.com/bndtools/bnd/blob/master/org.osgi.impl.bundle.repoindex.lib/src/org/osgi/service/indexer/impl/BundleAnalyzer.java#L418
and there are no standard way to get the host bsn ,see https://github.com/eclipse/birt/blob/bab46c1570900bf70d4ab750c8be3cc344553882/build/org.eclipse.birt.build/src/org/eclipse/birt/build/framework/Bundle.java#L170-L181

Comment (AndyWu2015) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/DeployCommand.java]: after you merge this , I will change the code in ide also.
